README.md

```diff
@@ -877,6 +877,51 @@ rmplan agent plan.yml --executor claude-code
 rmplan agent plan.yml --executor direct-call
 ```
 
+### Claude Code Executor: Interactive Tool Permissions
+
+The Claude Code executor includes an interactive permission system that allows you to control which tool invocations are automatically approved. When Claude attempts to use a tool during execution, you'll see a permission prompt with three options:
+
+- **Allow**: Permits this specific tool invocation only
+- **Disallow**: Denies this specific tool invocation
+- **Always Allow**: Permanently approves this tool (or command prefix for Bash tools) for automatic execution in future sessions
+
+#### Special Handling for Bash Commands
+
+The `Bash` tool receives special treatment due to its powerful nature. When you select "Always Allow" for a Bash command, an interactive prefix selection interface appears:
+
+1. The interface displays command tokens that you can navigate using arrow keys
+2. Press the right arrow to include more of the command in the approved prefix
+3. Press the left arrow to include less of the command
+4. Press 'a' to create a wildcard rule that approves all Bash commands (use with caution)
+5. Press Enter to confirm your selection
+
+For example, if Claude attempts to run `bun test src/utils.test.ts`, you can create rules like:
+
+- `bun test` - Approves all bun test commands
+- `bun` - Approves all bun commands
+- `*` - Approves all Bash commands (created by pressing 'a')
```

Comment:
Check that this is was 'a' does. It should select all the words in the current command, not set up a prefix that allows all commands.